test/e2e/operator.go

```diff
@@ -378,34 +385,108 @@ var _ = Describe("ARO Operator - Azure Subnet Reconciler", func() {
 })
 
 var _ = Describe("ARO Operator - MUO Deployment", func() {
-	It("must be deployed by default with FIPS crypto mandated", func(ctx context.Context) {
-		muoIsDeployed := func() (bool, error) {
-			By("getting MUO pods")
-			pods, err := clients.Kubernetes.CoreV1().Pods("openshift-managed-upgrade-operator").List(ctx, metav1.ListOptions{
-				LabelSelector: "name=managed-upgrade-operator",
-			})
-			if err != nil {
-				return false, err
-			}
-			if len(pods.Items) != 1 {
-				return false, fmt.Errorf("%d managed-upgrade-operator pods found", len(pods.Items))
-			}
+	When("we have a fresh ARO cluster", func() {
```

Comment:
Is this `When` needed? It doesn't check that the ARO cluster is fresh, and the same assumption is made in other operator e2e tests. I think you could follow the convention of the other operator e2e tests, and just use the `It`.

Reply:
Done.